### PR TITLE
BUGFIX: RAIL-2152 Document AttributeElements missing docs/api

### PR DESCRIPTION
**Pull request by Tulean**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/270/ : SUCCESS in 1m 11s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/118/ : SUCCESS in 5s

File: docs/attribute_element_option.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+All top-level options are optional. You can use only those that are relevant in your project.
+
+| Name | Required? | Type | Description |
+| :--- | :--- | :--- | :--- |
+| limit | false | number | Limit amount of elements |
+| offset | false | number | Offsets starting point of the element list |
+| order | false | [SortDirection](result_specification.md#sorting) | Ordering of the elements |
+| filter | false | string | Filter elements by text value |
+| uris | false | string[] | With this option you can specify concrete attribute elements to load. This is commonly used to preloaded selected elements in the attribute filter |
```
Comment (no23reason) [inline docs/attribute_element_option.md]: Detail: missing full stop at the end

File: docs/create_custom_attribute_filter.md [hunk 2]
```diff
@@ -102,3 +102,13 @@ export class AttributeElementsExample extends Component {
...
 ```
+
+## Properties
+
+| Name | Required? | Type | Description |
+| :--- | :--- | :--- | :--- |
+| projectId | true | string | The project ID |
+| sdk | false | SDK | A configuration object where you can define a custom domain and other API options |
+| uri | false | string | The attribute display form uri |
+| identifier | false | string | The attribute display form identifier |
```
Comment (no23reason) [inline docs/create_custom_attribute_filter.md]: We shoudl mention that people should only use uri OR identifier and that they should prefer identifiers.

Comment (no23reason) [reply docs/create_custom_attribute_filter.md]: Also, the example only works with uri, identifier is not passed anywhere.

**Comment by evgeniabogranskaya** [comment]
Please ping me when the tech review is done, I'll do a language review. Thanks!

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/272/ : SUCCESS in 53s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/120/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/276/ : SUCCESS in 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/124/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/277/ : SUCCESS in 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/125/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/278/ : SUCCESS in 46s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/79/ : SUCCESS in 16s

**Pull request merged**